crates/phala-types/src/lib.rs

```diff
@@ -601,6 +602,13 @@ pub struct WorkerRegistrationInfo<AccountId> {
     pub operator: Option<AccountId>,
 }
 
+#[derive(Encode, Decode, Debug, Clone, PartialEq, Eq, TypeInfo)]
+pub struct PhalaNetworkIdentBindingInfo {
+    pub pubkey: WorkerPublicKey,
+    pub pnetwork_ident: String,
```

Comment:
Let's use Vec<u8> instead. Otherwise it can make a lot of trouble in scale codec later

Reply:
Btw, I think the name of this field can be "endpoint"